docs/octopus-cloud/index.md

```diff
@@ -51,21 +51,52 @@ If you are having difficulty uploading packages to your Octopus built-in package
 
 In the near future, the Octopus Cloud will become available in additional regions and you'll have the ability to move between regions.
 
-## Reset the User Password
+## Reset the Octopus Account Password
+
+If you forget your username or password for your Octopus account or Octopus instance, you can reset them via Octopus.com
+
+1. Visit [Octopus.com/signin](https://Octopus.com/signin).
+1. Click 'Forgot your password?'
+1. Reset your credentials and log into your Octopus account. You can now launch your Octopus instance from the accounts dashboard.
+
+
+## Reset the Instance User Password
+:::hint
+This feature is being deprecated.
+:::
 
 If you forget your username or password for the instance, you can request a **new user invite**.
 
 1. Log in to your Octopus account.
 1. Select your cloud instance.
 1. Click the **new user invite** link.
 
-If you are trying to add new users to your Octopus Cloud instance, that is done in the instance itself. Learn more about [managing users and teams](/docs/administration/managing-users-and-teams/index.md).
+
+
+## Invite Users to your Octopus Cloud Instance {#OctopusCloud-Invitingusers}
+
+You can add new users to your Octopus Cloud instance from within your Octopus.com account. This feature is being rolled out gradually. If you have access to the invites feature, you will see invites in your instance panel. If you do not have access to this feature and would like access, please [contact support](https://octopus.com/support).
+
+1. Log into your account at [Octopus.com/signin](https://octopus.com/signin).
+1. Select **Manage** on your instance panel.
+1. Click **Invite a user**, enter their name and email address.
+1. Click **Submit**.
+
+If the user already has an [Octopus ID](/docs/administration/authentication/octopusid-authentication), they can **Sign in** to view their newly accessible instance within the correct Organization. Otherwise, they can **Register** for a new account with the email address the invitation was sent to.  All invitees will be set with the **Instance User** role. Users with the **User Instance** role will not be able to change any instance account settings or invite other users.
```

Comment:
```suggestion
If the user already has an [Octopus ID](/docs/administration/authentication/octopusid-authentication.md), they can **Sign in** to view their newly accessible instance within the correct Organization. Otherwise, they can **Register** for a new account with the email address the invitation was sent to.  All invitees will be set with the **Instance User** role. Users with the **User Instance** role will not be able to change any instance account settings or invite other users.
```